pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -947,9 +948,12 @@ func handleZbootRestarted(ctxArg interface{}, done bool) {
 	}
 }
 
-func handleInit(networkSendTimeout uint32) {
+
+//TODO: Complete this
+func handleInit(ctx *zedagentContext, networkSendTimeout uint32) {
 	initializeDirs()
 	handleConfigInit(networkSendTimeout)
+	ctx.configItemSpecMap.AddIntItem("timer.config.interval", 60, 5, 0xFFFFFFFF, false)
```

Comment:
You probably want to have 2 knobs here - globalSetting, agentSetting

Reply:
It can be both?

Reply:
Yes..